woeip/apps/air_quality/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+
+from woeip.apps.air_quality import models
+
+
+class SessionForm(forms.ModelForm):
```

Comment:
Given that you are only using `DustrakSessionForm`, I don't see much value in having this base class. Merge it for now, and extract it later if/when you needed.